modular_skyrat/modules/job_loadout_overrides/medical_doctor.dm

```diff
@@ -0,0 +1,2 @@
+/datum/outfit/job/doctor
```

Comment:
please put this in master_files or something; it doesn't need a whole module

Reply:
tomorrow